helix-view/src/document.rs

```diff
@@ -900,8 +900,9 @@ impl Document {
                     "Path is read only"
                 ));
             }
+            let write_path = std::fs::read_link(&path).unwrap_or_else(|_| path.clone());
```

Comment:
```suggestion
            let write_path = tokio::fs::read_link(&path).await.unwrap_or_else(|_| path.clone());
```
use the tokio function insteadt